### DISCUSSION TITLE
Implement the same program in PascalABC.NET?

### DISCUSSION THREAD

**Discussion by ghost** [pascalabcnet/pascalabcnet#3090] [Q&A] [answered]
Original SmallBASIC program:

https://github.com/smallbasic/SmallBASIC/blob/master/samples/distro-examples/graphics/3dtorus.bas

How to run: Download SmallBASIC release and run with `sbasicg -r 3dtorus.bas`

@SunSerega 

p/s: I don't need pointers on how to do it. I need a complete program as a demo to showcase it to other people. The purpose is to convince them that PascalABC.NET can do the same thing as SmallBASIC. Please help me. Thank you.

**Comment by SunSerega** [comment, marked as answer]
You can show many examples, including 3D graphics, from `C:\PABCWork.NET\Samples`
I was sure there was some similar example, also with a torus, drawn with lighting and all, without using the `Graph3D` unit. But now I can't find it...

But translating the thing you linked sounds like a lot of work with little payoff.
